From dc285d9a5dff7920de9ae9b0286116af965c9917 Mon Sep 17 00:00:00 2001
From: David Dillow <dillowda@ornl.gov>
Date: Sun, 16 Jan 2011 15:12:39 -0500
Subject: [PATCH 197/220] fix incorrect value of SCSI_MAX_SG_CHAIN_SEGMENTS due to include file ordering

If the compiled object doesn't include linux/scatterlist.h before
scsi/scsi.h, it will get an incorrect definition of
SCSI_MAX_SG_CHAIN_SEGMENTS.

Signed-off-by: David Dillow <dillowda@ornl.gov>
Cc: stable@kernel.org
Signed-off-by: James Bottomley <James.Bottomley@suse.de>
---
 include/scsi/scsi.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

--- a/include/scsi/scsi.h
+++ b/include/scsi/scsi.h
@@ -9,6 +9,7 @@
 #define _SCSI_SCSI_H
 
 #include <linux/types.h>
+#include <linux/scatterlist.h>
 
 struct scsi_cmnd;
 
